test/fixtures/keys/Makefile

```diff
@@ -27,6 +27,7 @@ ca3-key.pem:
 ca3-csr.pem: ca3.cnf ca3-key.pem
 	openssl req -new \
 		-extensions v3_ca \
+		-days 9999 \
```

Comment:
Gosh, this slipped my quick review. It isn't really necessary, since the days are set in `ca3.cnf`.

Reply:
Should we remove it or just leave it?

Reply:
Removed.